apps/dashboard/app/helpers/system_status_helper.rb

```diff
@@ -3,7 +3,7 @@
 # Helpers for the system status page /dashboard/systemstatus
 module SystemStatusHelper
   def title(cluster)
-    "#{cluster.metadata.title.to_s.titleize} Cluster Status"
```

Comment:
Actually now that I think about it, can we remove the `to_s` as it's unnessecary with the string interpolation `#{}`.